packages/vx-shape/src/shapes/BarGroup.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Group } from '@vx/group';
+import Bar from './Bar';
+import callOrValue from '../util/callOrValue';
+
+export default function BarGroup({
+  data,
+  className,
+  top,
+  left,
+  x0,
+  x0Scale,
+  x1Scale,
+  yScale,
+  zScale,
+  keys,
+  height,
+  ...restProps
+}) {
+  return (
+    <Group
+      className={cx('vx-bar-group', className)}
+      top={top}
+      left={left}
+    >
+      {data && data.map((d, i) => {
+        return (
+          <Group
+            key={`bar-group-${i}-${x0(d)}`}
+            left={x0Scale(x0(d))}
+          >
+            {keys && keys.map((key, i) => {
+              const value = d[key];
+              return (
+                <Bar
+                  key={`bar-group-bar-${i}-${value}-${key}`}
+                  x={x1Scale(key)}
+                  y={yScale(value)}
+                  width={x1Scale.bandwidth()}
+                  height={height - yScale(value)}
+                  fill={zScale(key)}
+                  {...Object.keys(restProps).reduce((ret, cur) => {
```

Comment:
this seems useful, could extract the entire function in addition to `callOrValue`